doc: add vignette (#10)
gladkia/vitorcid: vignettes/vitorcid.Rmd

library(vitorcid)
```

# vitorcid

The `vitorcid` package that allows you to generate a CV based on your ORCID profile.

# Installation

> 

Reviewer: Provide information regarding installation here, sth like:

(...)
You can install the package using `remotes` in the following way:
``` 
remotes::install_github("https://github.com/gladkia/vitorcid.git", ref = "main")
```
(...)

> Make installation in 3 steps:

Reviewer: Create a dedicated section from the 1st, 2nd, and 3rd points. Maybe `Authorization`?

> 1.  Make orcid authorization as described here: <https://github.com/ropensci-archive/rorcid/blob/master/vignettes/authentication.Rmd>

Reviewer: ```suggestion
1.  Authenticate with orcid.org as described [here](https://github.com/ropensci-archive/rorcid/blob/master/vignettes/authentication.Rmd)
```

> 2.  Create an environment variable 'ORCID TOKEN' with the token from point 2

Reviewer: ```suggestion
2.  Create an environment variable 'ORCID TOKEN'. Use the value obtained from the previous step.
```

```{r, eval = FALSE}
Sys.setenv(ORCID_TOKEN = "000abc")

```

> 3.  Create an environment variable with 'ORCID ID' with your ORCID ID

Reviewer: ```suggestion
3.  Create an environment variable with your 'ORCID ID'.
```

```{r, eval = FALSE}
Sys.setenv(ORCID_ID = "0000")

```

# CV generation - fast path

> 1.  Create account on <https://orcid.org/>

Reviewer: ```suggestion
1.  Create an account on <https://orcid.org/>.
```

> This is an example of a completed profile: HERE WILL BE MARYSIA'S PROFILE

Reviewer: ```suggestion
This is an example of a completed profile: [link](https://orcid.org/0000-0002-7059-6378/print).
```

> <https://orcid.org/0000-0002-7059-6378/print>

Reviewer: ```suggestion
```

**Employment** and **Education** sections are required.

**Websites** and **Social Links** are encouraged but not mandatory.

2.  Generate your CV:

```{r, eval = FALSE}
vitorcid::get_cv("CV")
```

> 3.  Review CV.pdf (generated CV) and edit generated template (CV.Rmd) at your own discretion.

Reviewer: ```suggestion
3.  Review CV.pdf (generated CV). If some corrections are needed you can do them in two ways:
* update the data at orcid.org and rerun `get_cv`
*  edit the template (CV.Rmd) and knit it (e.g. from within the Rstudio)
```
